dist/scripts/vendor.js

```diff
@@ -8062,7 +8062,7 @@ delete c[b.id], delete d[a];
 http:80,
 https:443,
 ftp:21
-}, sf = b("$location"), tf = /^\s*[\\/]{2,}/, uf = {
+}, sf = b("$location"), tf = /^\s*[\\\/]{2,}/, uf = {
```

Comment:
No, do you mind opening an issue? We either need to update the README or figure out what causes the diff with newer Node versions and fix it.